corehq/apps/reports/models.py

```diff
@@ -375,6 +375,30 @@ def query_groups(self):
             raise TableauAPIError("Error: API does not work with more than 1000 groups on a single site.")
         return response_body['groups']['group']
 
+    def _make_paginated_request_for_users(self, method_name, additional_url_path):
+        page_size = 100
+        page_number = 1
+        total_users = sys.maxsize
+        tableau_users = []
+        print(additional_url_path)
+        while (page_size * (page_number - 1) < total_users):
+            print(page_number)
+            response_body = self._make_request(
+                self.GET,
+                method_name,
+                (self.base_url
+                + f'{additional_url_path}?pageSize={page_size}&pageNumber={page_number}'),
+                {}
+            )
+            # If it's the first page, grab the total user count.
+            if page_number == 1:
+                total_users = int(response_body['pagination']['totalAvailable'])
```

Comment:
is there anything in here that tells you directly whether there are more pages available?